src/utils/misc-utils.ts

```diff
@@ -41,22 +41,18 @@ export const journalStatusIsError = (status: string | undefined) => {
 //  go/flowctl-go/cmd-discover.go
 export const MAX_DISCOVER_TIME = 30000;
 
-// TODO (optimization): Combine the stripPathing and truncateCatalogName utility functions.
-export const stripPathing = (stringVal: string) => {
+export const stripPathing = (stringVal: string, getTenant?: boolean) => {
```

Comment:
`tenantOnly` sounds like a better name for the optional parameter IMO.

Reply:
That makes sense - I didn't know what to name this one